CitationStyleLanguagePlugin.inc.php

```diff
@@ -416,6 +416,13 @@ public function getCitation($request, $article, $citationStyle = 'apa', $issue =
             $citationData->page = htmlspecialchars($publication->getData('pages'));
         }
 
+        // DOI
+        if (Repo::doi()->isEnabled()) {
+            if ($issue && $issue->getPublished()) {
+                $citationData->DOI = $article->getStoredPubId('doi');
```

Comment:
I reckon we don't even really need to check if dois are enabled. At a stage like this, if an article has a DOI, we should add it.